### DISCUSSION TITLE
django rest framework can use!

### DISCUSSION THREAD

**Discussion by katelei6** [apache/skywalking#8445] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

django2.2 use mysqlclient, not use pymysql, so ,skywalking cannot catch the metrics；we use django rest framework ，many rest api can not be collected！

### Use case

django rest framework
django2.2
mysqlclient

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Contributions are welcome. The plugins are built by the community.

**Comment by nongyue** [comment]
I can't use django rest framework with the same version mentioned

**Comment by nongyue** [comment]
> Contributions are welcome. The plugins are built by the community.

@wu-sheng is there some plugin making skywalking working with django rest framework? I did't find one!

**Comment by wu-sheng** [comment]
This is what we test about django, https://github.com/apache/skywalking-python/tree/master/tests/plugin/web/sw_django

**Comment by nongyue** [comment]
what if i must use fastapi (a web frame https://fastapi.tiangolo.com/), is there any plugin or something working with skywalking? @wu-sheng

**Comment by wu-sheng** [comment]
Then contribute a plugin. That is how open source works.
